pkg/blockatlas/platform.go

```diff
@@ -1,6 +1,7 @@
 package blockatlas
 
 import (
+	"github.com/trustwallet/blockatlas/db"
```

Comment:
in `RunSubscriber` for `types.DeleteSubscription` we need to remove all transactions for `to`, `from` when specified